pylabnet/utils/zi_hdawg_pulseblock_handler/zi_hdawg_pb_handler.py

```diff
@@ -0,0 +1,252 @@
+
+import numpy as np
+from pylabnet.utils.pulseblock.pb_sample import pb_sample
+
+
+# Sampling rate of HDWAG sequencer (300 MHZ)
+SEQ_SAMP_RATE = 300e6
```

Comment:
is this ever a variable parameter? can we change this? for example, if we want to run a very long and slow sequence it may be more memory efficient to reduce this, if we have that option in hardware

Reply:
We have great freedom over the sampling rate of the analogue waveforms, but the sequencer sampling rate is as far as I can tell not variable :(